src/devices/Mcp3xxx/Mcp33xx.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+
+namespace Iot.Device.Adc
+{
+    /// <summary>
+    /// MCP3304 Analog to Digital Converter (ADC)
+    /// </summary>
+    public abstract class Mcp33xx : Mcp30xx32xx
```

Comment:
nit: 33xx doesn't match either 30xx or 32xx consider renaming base class to `3xxx`